spark-connector/spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/iceberg/GravitinoIcebergCatalog.java

```diff
@@ -40,23 +39,18 @@ protected TableCatalog createAndInitSparkCatalog(
     return icebergCatalog;
   }
 
+  /**
+   * Migrated `loadTable(identifier)` to the BaseCatalog class and execute `loadTable(identifier)`
+   * before createSparkTable to load sparkTable with different parameters easily.
+   */
```

Comment:
this is more like a comment in PR not in code

Reply:
should i remove it?

Reply:
ok

Reply:
updated.